releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,16 @@
 
 [Image][image] | [Sample][sample]
 
+## 2.1.0 - 
+
+### Enhancements
+* Added support for load balancing mode, which enables your workloads to distribute requests to App Configuration across all available replicas. This enhancement improves the scalability of applications that typically experience high request volumes to App Configuration, ensuring they remain within quota limits. Load balancing mode is disabled by default and can be activated by setting `loadBalancingEnabled` to `true`. [#55](https://github.com/Azure/AppConfiguration-KubernetesProvider/issues/55)
+* Added support for refreshing configMap by watching all key-values. The Kubernetes provider will monitor the changes of selected key-values/feature flags and refresh the target ConfigMap only if the data updates. Watching sentinel key to refresh is still supported, it's optional now. [#40](https://github.com/Azure/AppConfiguration-KubernetesProvider/issues/40)
+* Added support for refreshing ConfigMaps and Secrets only if their data is really updated.

```

Comment:
Let's move this to bug fix:

Fix a bug where ConfigMaps and Secrets may be regenerated even if there is no data change.